components/org.wso2.carbon.identity.user.endpoint/src/gen/java/org/wso2/carbon/identity/user/endpoint/MeApiService.java

```diff
@@ -5,6 +5,7 @@
 
 import org.wso2.carbon.identity.user.endpoint.dto.ExportedUserDTO;
 import org.wso2.carbon.identity.user.endpoint.dto.SelfUserRegistrationRequestDTO;
+import org.wso2.carbon.identity.user.endpoint.dto.SuccessfulUserCreationDTO;
```

Comment:
Isn' this a redundant import?

Reply:
This is an auto-generated class. All the imports are automatically generated.